miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/WebViewDialog.kt

```diff
@@ -36,6 +38,17 @@ internal fun onShowDialog(
     return true
 }
 
+internal fun openNonSSLDialog(context: Context, url: String) =
+    AlertDialog.Builder(context)
+        .setMessage("This link is unsafe. If you would like to proceed, it will be opened in your native browser.")
+        .setPositiveButton(android.R.string.ok) { dialog, _ ->
+            context.startActivity(Intent(Intent.ACTION_VIEW, url.toUri()))
```

Comment:
I think, we need to handle `ActivityNotFoundException` when there is no browser is installed, otherwise it will cause issue.
We can just show a toast message?

```
            try {
                 ....
            } catch (e: ActivityNotFoundException) {
                Toast.makeText(context, "The browser hasn't been installed in this device.", Toast.LENGTH_LONG).apply {
                    setGravity(Gravity.BOTTOM, 0, 100)
                    show()
                }
            }
```